src/backend/distributed/Makefile

```diff
@@ -7,6 +7,8 @@ safestringlib_builddir = $(citus_top_builddir)/vendor/safestringlib/build
 safestringlib_a = $(safestringlib_builddir)/libsafestring_static.a
 safestringlib_sources = $(wildcard $(safestringlib_srcdir)/safeclib/*)
 
+CMAKE3 = $(shell which cmake3 || echo cmake)
```

Comment:
very minor: to be consistent maybe `$(shell which cmake3 || which cmake)`

Reply:
That's actually on purpose, I've added a comemnt now explaining why.

Reply:
Makes sense